cSpell.json

```diff
@@ -798,6 +798,18 @@
                 "Uknown"
             ]
         },
+        {
+            "filename": "**/specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/networkWatcher.json",
+            "words": [
+                "flowlogs"
+            ]
+        },
+        {
+            "filename": "**/specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/networkWatcher.json",
+            "words": [
+                "flowlogs"
```

Comment:
Hm, actually, why is it capitalized this way?

Reply:
I think you are referring to 'flowlogs'. it is this way from the beginning.